eth/protocols/snap/sync.go

```diff
@@ -1793,6 +1808,10 @@ func (s *Syncer) processStorageResponse(res *storageResponse) {
 					skipped++
 					continue
 				}
+				if _, err := res.overflow.Get(it.Key()); err == nil {
+					skipped++
+					continue
+				}
```

Comment:
Could you explain this? This change does looks like it modifies the basic snap sync algo - if so, is this something that should/could be applied on `master` already?

Reply:
Yes. So I'll resolve it.